src/main/java/com/google/gerrit/extensions/api/changes/ChangeApi.java

```diff
@@ -156,6 +156,12 @@ SubmittedTogetherInfo submittedTogether(
   @Deprecated
   void publish() throws RestApiException;
 
+  /** Set Code Review on a change*/
```

Comment:
These files in com.google name spaces are basically a copy of the official Gerrit API. I think you could move your implementation to ChangeEditApi? See: https://gerrit.googlesource.com/gerrit/+/master/java/com/google/gerrit/extensions/api/changes/ChangeEditApi.java

Reply:
No, because 'code review and verify', and 'submit' belong to 'Change' end point. They are not in the scope of 'ChangeEdit' end point

Reply:
I was wrong. Submit and review are part of RevisionApi. You should be able to use the existing methods.

Reply:
Correct, I will revert ChangeApi.java and ChangeApiRestClient.java

Reply:
done